### PR TITLE
enh: add unit test for get SG name

### PR DESCRIPTION
**Pull request by sakshi-1505**
**What this PR does / why we need it**:
Adds unit test for function in load balancer.go

**Which issue this PR fixes(if applicable)**:
refers https://github.com/kubernetes/cloud-provider-openstack/issues/2400

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/openstack/loadbalancer_test.go [hunk 1]
```diff
@@ -534,3 +535,48 @@ func Test_getListenerProtocol(t *testing.T) {
...
+				},
+			},
+			want: `lb-sg-test_uid_with_more_than_255_characters-test_namespace_with_more_than_255_characters_LoremipsumdolorsitametconsecteturadipiscingelitMaurissemperveldiamaposuereVivamustempusliberonecmaximuspellentesque-test_name_with_more_than_255_characters_Lo`,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := getSecurityGroupName(tt.args.service); got != tt.want {
+				t.Errorf("getSecurityGroupName() = %v, want %v", got, tt.want)
+			}
```
Comment (dulek) [inline pkg/openstack/loadbalancer_test.go:579]: You should use `assert.Equal` instead here.

**Review by dulek — CHANGES_REQUESTED** [review]
Seems like the test is failing. You can try them locally with `make unit`. Besides that and the inline comment this looks exactly like it should.

To be completely fair with the process #2389 was here first, so I'd rather merge the earlier one once it's finalized. Can you help with reviews on #2389 instead? Thanks!

**Comment by sakshi-1505** [comment]
>Seems like the test is failing. You can try them locally with make unit. Besides that and the inline comment this looks exactly like it should.

I actually did make unit before the pushing the PR through, looks like the spaces are creating issue.

**Comment by sakshi-1505** [comment]
@dulek I think it's pretty fair that we continue the discussion/changes on the PR opened by @KingDaemonX , I will be closing this PR in lieu of the existing PR https://github.com/kubernetes/cloud-provider-openstack/pull/2389

**Pull request closed without merging**